src/main/java/dev/luzifer/log4jexploitfix/AppenderFilter.java

```diff
@@ -45,6 +45,11 @@ public static void tryApply(java.util.logging.Logger logger) {
         public Result filter(LogEvent event) {
             
             String message = event.getMessage().getFormattedMessage();
+
+            if(message == null) {
+                return Result.NEUTRAL;
```

Comment:
Please use the implementation of the superclass
```suggestion
                return super.filter(event);
```